manifest.json

```diff
@@ -1,20 +1,28 @@
 {
-  "description": "A basic template for a Mozilla Rally Study.",
-  "author": "Mozilla",
+  "description": "Local News Demand Study",
+  "author": "Stanford GSB",
   "manifest_version": 2,
-  "name": "Rally Basic Study Template",
-  "version": "0.3.1",
-  "homepage_url": "https://github.com/mozilla-ion/ion-basic-study",
-
-  "applications": {
-    "gecko": {
-      "id": "ion-basic-study@mozilla.org",
-      "strict_min_version": "77.0a1"
-    }
-  },
-
-  "permissions": [],
+  "name": "Local News Demand Study",
+  "version": "0.1",
+  "homepage_url": "https://github.com/bchivers-stanford/local_news_demand",
 
+  "permissions": [
+    "<all_urls>",
+    "idle",
+    "notifications",
+    "storage",
+    "tabs",
+    "unlimitedStorage",
+    "webRequest",
+    "alarms",
+    "telemetry",
+    "history",
+    "webNavigation",
+    "webRequest",
```

Comment:
`webRequest` is duplicated here, the AMO linter alerted me :) It will not accept the extension as-is. No other errors though!

Reply:
Please also delete any unnecessary permissions here.

Reply:
Duplicate `webRequest` is deleted.  I did some testing and deleted permissions I believe are unnecessary (tabs, alarms, and history)